Client/src/Validation/validation.js

```diff
@@ -128,10 +128,43 @@ const maintenanceWindowValidation = joi.object({
 	monitors: joi.array().min(1),
 });
 
+const advancedSettingsValidation = joi.object({
+	apiBaseUrl: joi.string().uri({ allowRelative: true }).trim().messages({
+		"string.empty": "API base url is required.",
+		"string.uri": "The URL you provided is not valid.",
+	}),
+	logLevel: joi.string(),
```

Comment:
Acceptable values for `logLevel` are `["debug", "none", "error", "warn"]`

FYI for future FE validaiton you can also check the BE validation if it's a value that's being sent to the BE.  Validations are in the `./Server/validation/joi.js` file:

```
logLevel: joi.string().valid("debug", "none", "error", "warn").allow(""),
```

Reply:
fixed in https://github.com/bluewave-labs/bluewave-uptime/pull/986/commits/8ec2f2f16b916eb15cd5d092da768e29b55e6da2, thanks!